tests/tape/test_qscript.py

```diff
@@ -186,17 +186,153 @@ def test_update_par_info_update_trainable_params(self):
 
         p_i = qs._par_info
 
-        assert p_i[0] == {"op": ops[0], "p_idx": 0}
-        assert p_i[1] == {"op": ops[1], "p_idx": 0}
-        assert p_i[2] == {"op": ops[1], "p_idx": 1}
-        assert p_i[3] == {"op": ops[1], "p_idx": 2}
-        assert p_i[4] == {"op": ops[2], "p_idx": 0}
-        assert p_i[5] == {"op": ops[3], "p_idx": 0}
-        assert p_i[6] == {"op": ops[3], "p_idx": 1}
-        assert p_i[7] == {"op": m[0].obs, "p_idx": 0}
+        assert p_i[0] == {"op": ops[0], "op_idx": 0, "p_idx": 0}
+        assert p_i[1] == {"op": ops[1], "op_idx": 1, "p_idx": 0}
+        assert p_i[2] == {"op": ops[1], "op_idx": 1, "p_idx": 1}
+        assert p_i[3] == {"op": ops[1], "op_idx": 1, "p_idx": 2}
+        assert p_i[4] == {"op": ops[2], "op_idx": 2, "p_idx": 0}
+        assert p_i[5] == {"op": ops[3], "op_idx": 3, "p_idx": 0}
+        assert p_i[6] == {"op": ops[3], "op_idx": 3, "p_idx": 1}
+        assert p_i[7] == {"op": m[0].obs, "op_idx": 0, "p_idx": 0}
 
         assert qs._trainable_params == list(range(8))
 
+    def test_get_operation(self):
+        """Tests the fucntion get_operation."""
```

Comment:
```suggestion
        """Tests the function get_operation."""
```

Reply:
Wait, this test doesn't seem to call `get_operation`, does it?

Reply:
This test is a mistake of copy pasting, removed